### PR TITLE
DELETE a Boo! Deletes 2 pages! Adds more to "User" Page!

### PR DESCRIPTION
**Pull request by JustJeo**
- Delete button for BooThangs work (thanks guys)
- Deletes "Profile" Page
- Deletes "Messages" Page
- Changes [res.redirect] from "/profile" to "/user"

### CODE REVIEW

File: routes/user.js [hunk 1]
```diff
@@ -76,15 +76,23 @@ router.post("/messages", (req, res) => {
...
-router.delete("/", (req, res) => {
-  res.redirect("/");
+router.delete("/:id", (req, res) => {
+  db.boothang.destroy({
+      where: {
+        id: req.params.id
+      }
+    })
+    .then(deleteChick => {
+      console.log(`Bye~ ${deleteChick.name}` )
```
Comment (fostimus) [inline routes/user.js:93]: Heh

File: views/user/boothang.ejs [hunk 2]
```diff
@@ -7,7 +7,7 @@
   <ul class="boothangs">
     <% boothangs.forEach(function(boothang) { %>
       <li><%= boothang.name %> : <%= boothang.phoneNumber %></li>
-      <form action="/user/?_method=DELETE" method="post">
+      <form action="/user/<%= boothang.id %>?_method=DELETE" method="post">
         <!-- TODO, need some way to send unique value for boothang; primary key? phone number? -->
         <input type="hidden" name="phoneNumber" value="<%= boothang.phoneNumber %>">
```
Comment (fostimus) [inline views/user/boothang.ejs:12]: Since we aren't using this line, let's delete it! And the comment too :)

**Comment by fostimus** [comment]
Love it! Great stuff.

**Comment by JustJeo** [comment]
Just wanna let everyone know this push includes a lot of other stuff I was working on for the later half of the day. So don't forget to check out the extra commit too!

- Fixes the redirect of a timed out session
- Adds styling mainly to the "user" page
- Adds more pre-made messages for a user to choose from

**Pull request merged**